data/json/bionics.json

```
@@ -465,9 +465,9 @@
     "occupied_bodyparts": [ [ "torso", 20 ] ],
     "fuel_options": [ "battery" ],
     "fuel_efficiency": 1,
-    "fuel_capacity": 2000,
     "time": "1 s",
-    "flags": [ "BIONIC_POWER_SOURCE", "BIONIC_TOGGLED", "BIONIC_NPC_USABLE" ]
+    "flags": [ "BIONIC_POWER_SOURCE", "BIONIC_TOGGLED", "BIONIC_NPC_USABLE" ],
```

```suggestion
    "flags": [ "BIONIC_POWER_SOURCE", "BIONIC_TOGGLED" ],
```
Maybe those CBMs should lose the flag since NPCs can't reload them now?